pkg/types/validation/installconfig.go

```diff
@@ -1020,6 +1022,34 @@ func validateFeatureSet(c *types.InstallConfig) field.ErrorList {
 			if len(c.Compute) != 0 && c.Compute[0].Platform.VSphere != nil && len(c.Compute[0].Platform.VSphere.Zones) > 0 {
 				allErrs = append(allErrs, field.Forbidden(field.NewPath("compute", "platform", "vsphere", "zones"), errMsg))
 			}
+
+			if c.VSphere.LoadBalancer.Type != "" {
```

Comment:
I wanted to do:

```
if c.VSphere.LoadBalancer != nil {
  ...
}
```

But this doesn't work, I probably missed something obvious or is it because I'm not using a pointer?

Reply:
> because I'm not using a pointer

+1

Reply:
thanks